client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+import FoldableCard from 'components/foldable-card';
+import EllipsisMenu from 'components/ellipsis-menu';
+import PopoverMenuItem from 'components/popover/menu-item';
+import PopoverMenuSeparator from 'components/popover/menu-separator';
+import Gravatar from 'components/gravatar';
+
+class ActivityLogItem extends Component {
+
+	static propTypes = {
+		title: PropTypes.string,
+		subTitle: PropTypes.string,
+		className: PropTypes.string,
+		icon: PropTypes.string,
+		status: PropTypes.string,
+		user: PropTypes.object,
+		onClick: PropTypes.func,
+		actionText: PropTypes.string,
+		timestamp: PropTypes.number,
+		description: PropTypes.string
+	};
+
+	static defaultProps = {
+		onClick: noop,
+		status: 'is-info'
+	};
+
+	getTime() {
+		const {
+			moment,
+			timestamp
+		} = this.props;
+
+		return (
+			<div className="activity-log-item__time">
+				{ moment( timestamp ).format( 'LT' ) }
+			</div>
+		);
+	}
+
+	getIcon() {
+		const {
+			icon,
+			status
+		} = this.props;
+
+		const classes = classNames(
+			'activity-log-item__icon',
+			status
+		);
+
+		return (
+			<div className="activity-log-item__icons">
+				<div className={ classes }>
+					<Gridicon icon={ icon || 'info-outline' } size={ 24 } />
+				</div>
+			</div>
+		);
+	}
+
+	getActor() {
+		const {
+			user
+		} = this.props;
+
+		if ( ! user ) {
+			return null;
+		}
+
+		return (
+			<div className="activity-log-item__actor">
+				<Gravatar user={ user } size={ 48 } />
+				<div className="activity-log-item__actor-info">
+					<div className="activity-log-item__actor-name">{ user.name }</div>
+					<div className="activity-log-item__actor-role">{ user.role }</div>
+				</div>
+			</div>
+		);
+	}
+
+	getContent() {
+		const {
+			title,
+			subTitle
+		} = this.props;
+
+		return (
+			<div className="activity-log-item__content">
+				<div className="activity-log-item__content-title">{ title }</div>
+				{ subTitle && <div className="activity-log-item__content-sub-title">{ subTitle }</div> }
+			</div>
+		);
+	}
+
+	handleRestore = event => {
+		this.props.requestRestore( this.props.siteId, this.props.timestamp );
```

Comment:
if `requestRestore()` is going to be a redux action, it will need to mapped to a prop somethere, for example [like this](https://github.com/Automattic/wp-calypso/blob/75c13bd69894f2173a08f87c6cc6b8c18114c00a/client/my-sites/site-settings/custom-content-types/index.jsx#L167).

Maybe leave it out for now until the actions are built?